packages/api/src/external/fhir/document/index.ts

```diff
@@ -137,6 +137,14 @@ export const toFHIR = (
   });
 };
 
+export function createFHIRSubject(patientId: string): Reference<PatientFHIR> {
+  const subject: Reference<PatientFHIR> = {
+    reference: `Patient/${patientId}`,
+    type: "Patient",
+  };
+  return subject;
+}
```

Comment:
It would be good to have this on `packages/api/src/external/fhir/patient/index.ts` as `toFHIRSubject` or something